eq_translations/extractable_strings.py

```diff
@@ -239,6 +239,71 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..primary_content[*].title",
+        "description": "Introduction primary content",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].list[*]",
+        "description": "Introduction primary content list item",
+        "context": "PrimaryContent",
+        "additional_context": ["ListHeading", "ListDescription"],
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].description",
+        "description": "Introduction primary content description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].title",
+        "description": "Introduction primary content guidance title",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].description",
+        "description": "Introduction primary content guidance description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..preview_content.title",
+        "description": "Introduction preview content title",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.contents[*].description",
+        "description": "Introduction preview content description",
```

Comment:
Primary/Secondary/Preview content won't really make sense to a translator. Check with Donna, whether there is some other better context we can provide for these.

Reply:
Spoke to Donna and we made context easier to understand for the translator.